integration/db/src/main/resources/changelogs/changelog-2024-01-23-14-33-03.yml

```diff
@@ -0,0 +1,95 @@
+databaseChangeLog:
```

Comment:
I prefer using one changelog per version instead of "action" like these files unless we have a good reason to separate them. 
It helps reduce the number of files over time. Apart from that, the changesets per a pull request are closely related to each other. Combining the relevant changes in a file makes reading a bit easier.